props/saber_fett263_buttons.h

```diff
@@ -4290,8 +4309,12 @@ SaberFett263Buttons() : PropBase() {}
             handles_color_change |= current_config->blade##N->current_style() && current_config->blade##N->current_style()->IsHandled(HANDLED_FEATURE_CHANGE_TICKED);
             ONCEPERBLADE(USES_COLOR_CHANGE)
             if (!handles_color_change) {
-              current_preset_.Save();
               color_mode_ = CC_COLOR_LIST;
+              show_color_all_.Start();
+              for (int i = 1; i <= NUM_BLADES; i++) {
+                ShowColorStyle::SetColor(GetColorArg(i, BASE_COLOR_ARG));
+                if (style_parser.UsesArgument(current_preset_.GetStyle(i), BASE_COLOR_ARG + 2)) break;
```

Comment:
This seems weird to me.
It breaks as soon as it finds a style that doesn't use the given argument.
Since the blades could be in any order, it seems odd to have one style change how other styles work.

Reply:
Yes, I didn't think we could have multiple colors in ShowColorAllBladesTemplate<> so I have it use the first color argument it finds.  Is that not correct?  How would we have the blades individually show their current color is they are all using show_color_all_?

Reply:
Hmm, I think the code is probably correct, just confusing.
I think it should probably look like this:

for (....) {
  if (  ... UsesArgument( )) {
     ShowColorStyle::SetColor(....)
     break;
  }
}

Reply:
ok, updated, thanks.